packages/sns/src/converters/governance.converters.ts

```diff
@@ -378,6 +384,7 @@ export const fromCandidAction = (action: ActionCandid): Action => {
     return { Motion: action.Motion };
   }
 
+  // TODO: Find a better way to log this because JSON.stringify doesn't support BigInt.
```

Comment:
maybe something like:
```
JSON.stringify(action, (key, value) =>
  typeof value === 'bigint' ? value.toString() : value
);
```

Reply:
Looks good, let's do it in a separate pr 👍

Reply:
Just sharing if interesting: we already have utils for this.

See [jsonReplacer](https://github.com/dfinity/ic-js/tree/main/packages/utils#gear-jsonreplacer) and [jsonReviver](https://github.com/dfinity/ic-js/tree/main/packages/utils#gear-jsonreviver) of `@dfinity/utils`.